plugins/database/src/db_plugin.cpp

```diff
@@ -145,47 +147,64 @@ namespace
 
             for (auto result = nanodbc::execute(stmt); result.next();) {
                 const auto option_name = result.get<std::string>(0);
-
-                if (option_name == irods::KW_CFG_PAM_PASSWORD_MIN_TIME) {
-                    _out.password_min_time = std::stoll(result.get<std::string>(1));
-                    if (_out.password_min_time < 0) {
-                        return ERROR(CONFIGURATION_ERROR,
-                                     fmt::format("Invalid configuration for option [{}] in namespace [{}]: [{}]",
-                                                 option_name,
-                                                 _namespace,
-                                                 _out.password_min_time));
+                const auto option_value = result.get<std::string>(1);
+
+                if (option_name == irods::KW_CFG_PAM_PASSWORD_MIN_TIME ||
+                    option_name == irods::KW_CFG_PAM_PASSWORD_MAX_TIME) {
+                    const bool min = (option_name == irods::KW_CFG_PAM_PASSWORD_MIN_TIME);
```

Comment:
Consider finding a way to do the following string comparison once.
```
option_name == irods::KW_CFG_PAM_PASSWORD_MIN_TIME
```

Reply:
Done, I think

Reply:
I would also like to point out that this section is the part I like least of these changes, but I couldn't think of a better way to do it in this timeframe nor do I think it's worth putting much more effort into. Maybe that's just me :)

Reply:
Yep, understood. It passes the tests and hopefully no one has to touch this chunk of code again for a long time.